lib/vericite.rb

```diff
@@ -263,7 +263,7 @@ def sendRequest(command, args)
           external_content_data.upload_content_length = args[:psize]
           report_meta_data.external_content_data = external_content_data
           # @return [Array<ExternalContentUploadInfo>]
-          data, status_code, headers = vericite_client.reports_submit_request_context_id_assignment_id_user_id_post(context_id, assignment_id, user_id, consumer, consumer_secret, report_meta_data)
+          data, status_code, headers = vericite_client.reports_submit_request_context_id_assignment_id_user_id_post(context_id, assignment_id, user_id, consumer, consumer_secret, report_meta_data, {:'encrypted' => true})
```

Comment:
`{ encrypted: true }` should be the format for this hash.

Also, if that is going to be hard coded, is this something the vericite gem should be handling?  Is there ever a time where you wouldn't want it encrypted?